src/Models/MagentoGroup.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace JustBetter\MagentoPrices\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Carbon;
+
+/**
+ * @property int $id
+ * @property string $code
+ * @property array $data
+ * @property ?Carbon $imported_at
+ * @property ?Carbon $created_at
+ * @property ?Carbon $updated_at
+ */
+class MagentoGroup extends Model
```

Comment:
Maybe `MagentoProductGroup` is a more suitable name

Reply:
I'd choose `MagentoCustomerGroup` ;)